install/kubernetes/helm/istio/charts/tracing/templates/service-jaeger.yaml

```diff
@@ -0,0 +1,79 @@
+{{ if .Values.jaeger.enabled }}
+
+apiVersion: v1
+kind: List
+items:
+- apiVersion: v1
+  kind: Service
+  metadata:
+    name: jaeger-query
+    namespace: {{ .Release.Namespace }}
+    labels:
+      app: jaeger
+      jaeger-infra: jaeger-service
+      chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+      release: {{ .Release.Name }}
+      heritage: {{ .Release.Service }}
+  spec:
+    ports:
+      - name: query-http
+        port: 80
+        protocol: TCP
+        targetPort: {{ .Values.service.uiPort }}
+    selector:
+      app: jaeger
+    type: LoadBalancer
```

Comment:
should have been `{{ .Values.service.type }}`, fixed in https://github.com/istio/istio/pull/5760